packages/driver/cypress/integration/e2e/origin/config_env.spec.ts

```diff
@@ -10,14 +10,17 @@
       error: 'cy-origin-error',
     }
 
-    beforeEach(() => {
-      cy.visit('/fixtures/primary-origin.html')
-      cy.get('a[data-cy="cross-origin-secondary-link"]').click()
-    })
-
     afterEach(() => {
       // @ts-ignore
       window.top.__cySkipValidateConfig = true
+
+      // FIXME: Tests that end with a cy.origin command and enqueue no further cy
+      // commands may have origin's unload event bleed into subsequent tests
+      // and prevent stability from being reached, causing those tests to hang.
+      // We enqueue another cy command after each test to ensure stability
+      // is reached for the next test. This additional command can be removed with the
+      // completion of: https://github.com/cypress-io/cypress/issues/21300
+      cy.then(() => { /* ensuring stability */ })
```

Comment:
We still need this even after removing the before each? I wonder if there is another problem lurking here.

Reply:
I'll confirm whether or not this is still necessary with the removal of the beforeEach. At one point I thought it was, but local runs seem to pass without it now, could have been an error on my part.

Reply:
Can confirm it's not necessary, thanks for calling me on this one.